CONTRIBUTING.md

```diff
@@ -176,7 +176,7 @@ Start elasticsearch from a nightly snapshot.
 yarn es snapshot
 ```
 
-This will run Elasticsearch with a `trial` license and bootstraped with the user `elastic` and password `changeme`. Additional options are available, pass `--help` for more information.
```

Comment:
Do we mention these credentials anywhere else so folks know what they need when security is enabled?